### PR TITLE
Add init/cleaup to waltest to make it work

### PR DESCRIPTION
**Pull request by junli1026**

### CODE REVIEW

File: src/wal/test/waltest.c [hunk 1]
```diff
@@ -36,7 +37,7 @@ int writeToQueue(void *pVnode, void *data, int type, void *pMsg) {
 }
 
 int main(int argc, char *argv[]) {
-  char path[128] = "/home/jhtao/test/wal";
+  char path[128] = {0};
```
Comment (stephenkgu) [inline src/wal/test/waltest.c]: We need a non null default value here for help (./waltest -h) printing.
Any other fixes look good to me.

Comment (junli1026) [reply src/wal/test/waltest.c]: sure, will fix.

Comment (junli1026) [reply src/wal/test/waltest.c]: I put "/tmp/wal" as default value in the second iteration.

**Review by stephenkgu — APPROVED** [review]

**Review by sangshuduo — APPROVED** [review]
LGTM

**Review by guanshengliang — APPROVED** [review]

**Pull request merged**